e2e/specs/openConceptLab.spec.ts

```diff
@@ -13,22 +13,26 @@ test.beforeEach(async ({ api }) => {
 test('should be able to setup a subscription and import concepts', async ({ page }) => {
```

Comment:
Test name should be renamed with the action. Check other repos.

Reply:
@jayasanka-sack Sure!